WooCommerce/Classes/ViewRelated/Blaze/AdDestination/BlazeAdDestinationSettingView.swift

```diff
@@ -0,0 +1,222 @@
+
+                    VStack {
+                        ForEach(viewModel.parameters, id: \.self) { parameter in
+                            parameterItem(itemName: parameter.key)
+                        }
+
+                        Button(Localization.addParameterButton) {
+                            // todo
+                        }
+                        .buttonStyle(PlusButtonStyle())
+                        .padding([.leading, .trailing], Layout.contentSpacing)
+                        .padding(.bottom, Layout.parametersVerticalSpacing)
+                    }
+                    .background(Color(.systemBackground))
+                }
+
+                // Remaining characters and final destination
+                VStack(alignment: .leading) {
+                    Text(viewModel.remainingCharactersLabel)
+                        .subheadlineStyle()
+                        .padding(.bottom, Layout.contentVerticalSpacing)
+
+                    Text(viewModel.finalDestinationLabel)
+                        .subheadlineStyle()
+                }
+                .padding([.leading, .trailing], Layout.contentSpacing)
+                .padding([.top, .bottom], Layout.contentVerticalSpacing)
+                .background(Color(.systemGray6))
+
+                Spacer()
+            }
+            .frame(maxWidth: .infinity, alignment: .leading)
+            .background(Color(.systemGray6))
+            .navigationBarTitleDisplayMode(.inline)
+            .navigationTitle(Localization.adDestination)
+            .toolbar {
+                ToolbarItem(placement: .cancellationAction) {
+                    Button(Localization.cancel) {
+                        dismiss()
+                    }
+                }
+            }
+        }
+    }
+    private func sectionHeading(title: String) -> some View {
+        Text(title.uppercased())
+            .subheadlineStyle()
+            .frame(maxWidth: .infinity, alignment: .leading)
+            .padding(Layout.sectionHeadingPadding)
+    }
+
+    private func destinationItem(title: String,
+                                 subtitle: String,
+                                 type: DestinationType,
+                                 showBottomDivider: Bool = false) -> some View {
+        HStack(alignment: .center) {
+            if type == viewModel.selectedDestinationType {
+                Image(systemName: "checkmark")
+                    .padding(.leading, Layout.contentSpacing)
+                    .padding(.trailing, Layout.contentHorizontalSpacing)
+                    .foregroundColor(Color(uiColor: .accent))
+            } else {
+                Image(systemName: "checkmark")
+                    .padding(.leading, Layout.contentSpacing)
+                    .padding(.trailing, Layout.contentHorizontalSpacing)
+                    .hidden() // Small hack to make the icon space consistent while not showing the icon.
+            }
```

Comment:
This can be simplified with the custom `if` modifier:

```suggestion
            Image(systemName: "checkmark")
                .padding(.leading, Layout.contentSpacing)
                .padding(.trailing, Layout.contentHorizontalSpacing)
                .if(type != viewModel.selectedDestinationType) { view in
                    view.hidden()
                }
```

Reply:
Oh nice! TIL about the if modifier.